test/types/string/kbrady/multilocale/strided-search.future

```diff
@@ -0,0 +1 @@
+unimplemented feature:  string slicing by stridable codepoint ranges
```

Comment:
Is it possible to also add a `.bad` file for this (and other) futures so that we know when the behavior changes or it starts breaking for reasons other than the current one?

Reply:
Yes, I can do that.